skbio/metadata/mixin.py

```diff
@@ -0,0 +1,519 @@
+# ----------------------------------------------------------------------------
```

Comment:
you didn't pull the latest from master branch? This is renamed to skbio/metadata/_mixin.py